pkg/web/templates/partials/secure_envelope/secure_envelope_detail.html

```diff
@@ -16,7 +16,96 @@ <h3>Outgoing:</h3>
     <span>ID: {{ .ID }}</span>
   </div>
   <div class="collapse-content">
-    <div class="mt-4">
+    {{ $originatingVasp := .Identity.OriginatingVasp.OriginatingVasp.Person.LegalPerson }}
+    {{ $beneficiaryVasp := .Identity.BeneficiaryVasp.BeneficiaryVasp.Person.LegalPerson }}
+    <section class="mt-4">
+      <h3 class="font-semibold">Virtual Asset Service Providers (Legal Persons)</h3>
```

Comment:
I attempted to add a nested accordion, but that didn't work as expected with `daisyUI`. I thought it would be better to at least display the info and then we can figure out how to either nest accordions with a different library or consider a different style for this page.

Reply:
That makes sense - also I don't know why this is making me thing of the accept/repair form - but perhaps we should do something similar in those forms where we just display the VASP info side by side and not make it editable?